### DISCUSSION TITLE
夜间模式适配问题

### DISCUSSION THREAD

**Discussion by Auroradanica** [the1812/Bilibili-Evolved#3212] [提问 / Q&A] [answered]
![image](https://user-images.githubusercontent.com/87475428/161571923-a40e2cca-90e8-4197-877d-79dbb8ba8912.png)
在分区的页面夜间模式很奇怪，求解决

**Comment by Geeyun-JY3** [comment, marked as answer]
请先善用搜索，重复：#3193
